coalib/settings/Section.py

```diff
@@ -36,14 +39,50 @@ def append_to_sections(sections,
     if not section_name.lower() in sections:
         sections[section_name.lower()] = Section(section_name)
 
-    sections[section_name.lower()].append(
-        Setting(key, str(value), origin, from_cli=from_cli))
+    sections[section_name.lower()].append(Setting(
+        key, str(value), origin, to_append=to_append, from_cli=from_cli))
 
 
 @generate_repr()
 class Section:
     """
     This class holds a set of settings.
+
+    To add settings and sections to a dictionary of sections we can use
+    `append_to_sections`:
+
+    >>> sections = {}
+    >>> append_to_sections(sections,
+    ...                    'test1',
+    ...                    'val',
+    ...                    'origin',
+    ...                    section_name='default')
+    >>> 'default' in sections
+    True
+    >>> len(sections)
+    1
+    >>> len(sections['default'].contents)
+    1
```

Comment:
maybe just print the `repr` of the `sections` here and use `...` in the right places to make it well readable